docset/winserver2022-ps/activedirectory/Get-ADComputer.md

```diff
@@ -11,52 +11,71 @@ title: Get-ADComputer
- [-ResultPageSize <Int32>] [-ResultSetSize <Int32>] [-SearchBase <String>] [-SearchScope <ADSearchScope>]
- [-Server <String>] [<CommonParameters>]
+Get-ADComputer [-AuthType <ADAuthType>] [-Credential <PSCredential>] 
+  -Filter  <String> [-Properties <String[]>] [-ResultPageSize <Int32>]
+  [-ResultSetSize <Int32>] [-SearchBase <String>] [-SearchScope <ADSearchScope>]
+  [-Server <String>] [<CommonParameters>]
 ```
 
 ### Identity
+
 ```powershell
-Get-ADComputer [-AuthType <ADAuthType>] [-Credential <PSCredential>] [-Identity] <ADComputer>
- [-Partition <String>] [-Properties <String[]>] [-Server <String>] [<CommonParameters>]
+Get-ADComputer [-AuthType <ADAuthType>] [-Credential <PSCredential>] 
+  [-Identity] <ADComputer> [-Partition <String>] [-Properties <String[]>]
+  [-Server <String>] [<CommonParameters>]
 ```
 
 ### LdapFilter
+
 ```powershell
 Get-ADComputer [-AuthType <ADAuthType>] [-Credential <PSCredential>] -LDAPFilter <String>
- [-Properties <String[]>] [-ResultPageSize <Int32>] [-ResultSetSize <Int32>] [-SearchBase <String>]
- [-SearchScope <ADSearchScope>] [-Server <String>] [<CommonParameters>]
+  [-Properties <String[]>] [-ResultPageSize <Int32>] [-ResultSetSize <Int32>]
+  [-SearchBase <String>] [-SearchScope <ADSearchScope>] [-Server <String>] 
+  [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Get-ADComputer** cmdlet gets a computer or performs a search to retrieve multiple computers.
-
-The *Identity* parameter specifies the Active Directory computer to retrieve.
-You can identify a computer by its distinguished name, GUID, security identifier (SID) or Security Accounts Manager (SAM) account name.
-You can also set the parameter to a computer object variable, such as `$<localComputerobject>` or pass a computer object through the pipeline to the *Identity* parameter.
 
-To search for and retrieve more than one computer, use the *Filter* or *LDAPFilter* parameters.
-The *Filter* parameter uses the PowerShell Expression Language to write query strings for Active Directory.
-PowerShell Expression Language syntax provides rich type conversion support for value types received by the *Filter* parameter.
-For more information about the *Filter* parameter syntax, type `Get-Help` [about_ActiveDirectory_Filter](/previous-versions/windows/server/hh531527(v=ws.10)).
-If you have existing Lightweight Directory Access Protocol (LDAP) query strings, you can use the *LDAPFilter* parameter.
+The **Get-ADComputer** cmdlet gets a computer or performs a search to retrieve multiple computers.
 
-This cmdlet retrieves a default set of computer object properties.
-To retrieve additional properties use the *Properties* parameter.
-For more information about the how to determine the properties for computer objects, see the *Properties* parameter description.
+The _Identity_ parameter specifies the Active Directory computer to retrieve.
```

Comment:
We always bold our parameter names when they're not prefixed with a `-`, like `-Identity`. When they're prefixed, we wrap them in backticks to render as code.

```suggestion
The **Identity** parameter specifies the Active Directory computer to retrieve.
```